test/e2e/linux/test_job_submissions.py

```diff
@@ -43,7 +52,7 @@ def test_success(
 
         # THEN
         LOG.info(f"Waiting for job {job.id} to complete")
-        job.wait_until_complete(client=deadline_client)
+        job.wait_until_complete(client=deadline_client, max_retries=20)
```

Comment:
Would you mind looking into adding a non-`None` default to the [`wait_until_complete` implementation](https://github.com/aws-deadline/deadline-cloud-test-fixtures/blob/1f27578da0f2fe9cc241334acd439d9e1d741a1d/src/deadline_test_fixtures/deadline/resources.py#L577-L583)? The default, right now, is infinite retries forever. That's dangerous in that an unexpected malfunction would mean that automated tests will literally run forever, or until we take notice and nuke the test runner. It can be a really big default, but there should still be a default, imo.

Reply:
Agreed, I'll add it to the implementation.

Reply:
https://github.com/aws-deadline/deadline-cloud-test-fixtures/pull/134